packages/create-app/src/createApp.ts

```diff
@@ -65,9 +65,17 @@ export default async (opts: OptionValues): Promise<void> => {
     },
   ]);
 
+  // Use `default-app-next` as default template when `--experimental-frontend` is specified.
+  // Otherwise, use `default-app`.
+  const defaultTemplatePath = opts.experimentalFrontend
+    ? 'templates/default-app-next'
+    : 'templates/default-app';
+
+  // Use `--template-path` argument as template when specified.
+  // Otherwise, use the default template.
   const templateDir = opts.templatePath
     ? paths.resolveTarget(opts.templatePath)
-    : paths.resolveOwn('templates/default-app');
+    : paths.resolveOwn(defaultTemplatePath);
```

Comment:
There's a lot of duplication due to most of the template being identical. Afaik all we need is a different `packages/app` template. Could you explore if it's simple enough to only have a template for the `app` package instead? and use the existing template for the rest.